Earthfile

```diff
@@ -106,6 +106,8 @@ version:
 
 build-kairos-agent:
     FROM +go-deps
+    BUILD +webui-deps
```

Comment:
you shouldn't need to specify BUILD. `COPY` should automatically make it build the deps

Reply:
@mudler indeed it is not needed for the building of the agent, but if I don't keep it then, it is missing from my local

Reply:
oh ok I see! otoh I'd be rather surprised by the build command to populate something in my source code, maybe it's just ok to let that to be called separately; I don't see us touching those and maybe just gets in the way

Reply:
@mudler oki removed it